### PR TITLE
[15.0][MIG]-Account_move_line_purchase_info

### PR DESCRIPTION
**Pull request by JoanMForgeFlow**
Standard migration to version 15: module Account move line purchase info

### CODE REVIEW

**Review by mariadforgeflow — APPROVED** [review]
LGTM! :)

**Review by BernatPForgeFlow — APPROVED** [review]

File: account_move_line_purchase_info/tests/test_account_move_line_purchase_info.py [hunk 1]
```diff
@@ -0,0 +1,225 @@
+# Copyright 2019 ForgeFlow S.L.
+#   (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields
+from odoo.tests import Form, common
+
+
+class TestAccountMoveLinePurchaseInfo(common.TransactionCase):
+    def setUp(self):
```
Comment (ivantodorovich) [inline account_move_line_purchase_info/tests/test_account_move_line_purchase_info.py]: Use `setUpClass` instead

File: account_move_line_purchase_info/models/purchase_order_line.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Copyright 2019-2020 ForgeFlow S.L.
+#   (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class PurchaseOrder(models.Model):
```
Comment (ivantodorovich) [inline account_move_line_purchase_info/models/purchase_order_line.py]: I'd move this to a separate `purchase_order.py` file

File: account_move_line_purchase_info/models/purchase_order_line.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    _inherit = "purchase.order"
+
+    @api.depends("order_line.invoice_lines.move_id")
+    def _compute_invoice(self):
+        """Overwritten compute to avoid show all Journal Entries with
+        purchase_order_line as invoice_lines One2many would take them into account."""
+        for order in self:
+            invoices = order.mapped("order_line.invoice_lines.move_id").filtered(
```
Comment (ivantodorovich) [inline account_move_line_purchase_info/models/purchase_order_line.py]:
```suggestion
            invoices = order.order_line.invoice_lines.move_id.filtered(
```
mapping is not necessary here, anyway not blocking

File: account_move_line_purchase_info/models/purchase_order_line.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+                lambda m: m.is_invoice(include_receipts=True)
+            )
+            order.invoice_ids = [(6, 0, invoices.ids)]
```
Comment (ivantodorovich) [inline account_move_line_purchase_info/models/purchase_order_line.py]:
```suggestion
            order.invoice_ids = invoices
```
same but looks better :P

Comment (JoanMForgeFlow) [reply account_move_line_purchase_info/models/purchase_order_line.py]: @ivantodorovich Thanks for the review! Solved :)

**Review by ivantodorovich — APPROVED** [review]

File: account_move_line_purchase_info/__manifest__.py [hunk 5]
```diff
@@ -5,7 +5,7 @@
 {
     "name": "Account Move Line Purchase Info",
     "summary": "Introduces the purchase order line to the journal items",
-    "version": "14.0.1.0.1",
+    "version": "15.0.1.0.1",
```
Comment (AaronHForgeFlow) [inline account_move_line_purchase_info/__manifest__.py]: 15.0.1.0.0

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]
Functional review :+1:  Just a  minor comment

**Review by AaronHForgeFlow — APPROVED** [review]

**Pull request merged**